app/components/buttons/tab-button.jsx

```
@@ -0,0 +1,12 @@
+import { twMerge } from "tailwind-merge";
+
+
+export default function TabButton({className = "", label, isActive, onClick, hasIcon = false, icon = "" }) {
+    return (
+        <button className={twMerge(`tab-button px-5 py-2 cursor-pointer ${isActive ? 'active border-b-2 border-[--md-sys-color-primary]' : ''} relative`, className)} onClick={onClick}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)